API/index.js

```diff
@@ -172,9 +172,20 @@ class API {
 
       /* -------------------------- Error Handling ---------------------------*/
       if (status === 429) {
+        // Case-insensitive header lookup
+        const [rateLimitTypeHeader] = Object.keys(headers).filter(
+          (key) => { return (key.toLowerCase() === 'x-ratelimit-type'); }
+        );
+        if (!headers[rateLimitTypeHeader]) {
+          // Missing rate limit type header
+          throw new ZACCLError({
+            message: 'Zoom is very busy right now. Please try this operation again later.',
+            code: ERROR_CODES.UNKNOWN_LIMIT_ERROR,
+          });
```

Comment:
@gemmaw531 From my tests, the Zoom API Status:200 responses don't include 'x-ratelimit-type'

Example from rate tests from a http200 and http429 
`
info:   #51 status:200 x-ratelimit-category:Medium, date:Tue, 18 Aug 2020 15:22:31 GMT
info:   #53 status:429 (body-code:429) x-ratelimit-category:Medium, x-ratelimit-type:QPS, message:You have reached the maximum per-second rate limit for this API. Try again later., date:Tue, 18 Aug 2020 15:22:31 GMT
`

Reply:
:)